src/backend/renderer.js

```diff
@@ -478,7 +478,7 @@ export function attach(
           return elementType.displayName;
         } else {
           const displayName = type.displayName || type.name;
-          return displayName ? `Memo(${displayName})` : 'Memo';
+          return displayName ? displayName : 'Memo';
```

Comment:
I think we might want to change the default name in these cases now to be "Unknown" (since we'll being showing a badge next to it with "Memo" or "ForwardRef"